index.bs

```diff
@@ -1550,7 +1554,7 @@ session.CapabilityRequest = {
   ? browserVersion: text,
   ? platformName: text,
   ? proxy: session.ProxyConfiguration,
-  ? webSocketUrl: bool,
+  ? userPromptHandler: session.UserPromptHandler,
```

Comment:
The capability is named `unhandledPromptBehavior`.

Reply:
```suggestion
  ? unhandledPromptBehavior: session.UserPromptHandler,
```